src/signing/mod.rs

```diff
@@ -29,6 +32,12 @@ pub enum SignerType {
     #[cfg(feature = "stronghold")]
     #[cfg_attr(docsrs, doc(cfg(feature = "stronghold")))]
     Stronghold,
+    /// Ledger Device
+    #[cfg(feature = "ledger-nano")]
+    LedgerNanoSigner,
```

Comment:
we should probably rename these to just `LedgerNano` and `LedgerNanoSimulator`, it feels redundant to have the "Signer" suffix

Reply:
ok :) PR is still open, so I'll change it - and I also have to push a fix for the bip32 inputs that I missed to sort :see_no_evil: